cowprotocol/cowswap
# feat(tx-bundling): analytics

# Summary

Adds analytics for the 2 new tx bundling flows: swap and limit

# To Test

1. Install [GA debugger extension](https://chrome.google.com/webstore/detail/google-analytics-debugger/jnkmfdileelhofjcijamephohjechhna?hl=en) and turn it on
![image](https://github.com/cowprotocol/cowswap/assets/43217/f91df5bc-6d42-4044-b1d9-fed969187219)

2. Open the PR inside the Safe as a custom app
3. Open the dev console and filter for `ga`
4. Place Swap/Limit order using bundle tx
* The event named `Bundle Approve and Swap` should be logged right after the Safe pop up is displayed
5. Submit the safe tx
* The event named `Sign` should be logged after the Safe modal closes
